lib/davinci_dtr_test_kit/tags.rb

```diff
@@ -5,4 +5,8 @@ module DaVinciDTRTestKit
   CLIENT_NEXT_TAG = 'dtr_questionnaire_next_question'.freeze
   EHR_AUTHORIZE_TAG = 'dtr_smart_app_ehr_authorize'.freeze
   SUPPORTED_PAYER_TAG = 'light_ehr_supported_payer_endpoint'.freeze
+  REGISTRATION_TAG = 'registration'.freeze
+  TOKEN_TAG = 'token'.freeze
+  UDAP_TAG = 'udap'.freeze
+  SMART_TAG = 'smart'.freeze
```

Comment:
Unused?

Reply:
removed - the tags defined in the SMART and UDAP test kits are used instead